cross_list_courses/migrations/0001_initial.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql = "INSERT INTO lti_permissions_ltipermission "
+                  "(permission, school_id, canvas_role, allow) "
+                  "VALUES ('cross_listing', '*', 'AccountAdmin', TRUE),"
+                  "('cross_listing', '*', 'Account Admin', TRUE);",
```

Comment:
Hmm, note sure how it works  for me without that.. I have only 2 records in my local BD. But I will add it for now and chase it down later.